build.gradle.kts

```diff
@@ -22,9 +22,9 @@ repositories {
 }
 
 dependencies {
-  implementation("org.sonarsource.scanner.gradle:sonarqube-gradle-plugin:2.8")
-  implementation("org.jlleitschuh.gradle:ktlint-gradle:9.2.1")
-  implementation("io.gitlab.arturbosch.detekt:detekt-gradle-plugin:1.6.0")
+  implementation("org.sonarsource.scanner.gradle:sonarqube-gradle-plugin:3.0")
```

Comment:
We've had some breaking changes in the ordering of a few tasks with this change. I think @fpavageau was looking into it

Reply:
The plugin isn't using it directly. But you're right the consumers should care.

Reply:
@celcius112 Next release will be 2.0 so it's ok if we have breaking changes.

Reply:
thanks